src/bilder/images/edxapp/files/migrations.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description="Obtain an exclusive lock and run database migrations at startup."
+Requires=network-online.target
+After=consul-template.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/bin/consul lock -timeout=1h -verbose lsm_migrations /edx/bin/edxapp-migrate-lms
```

Comment:
nit: typo. `lsm_migrations` -> `lms_migrations`

Reply:
It might be better to set a lock timeout longer than 1 hour to account for the potential of migrations to take a long time on large tables. It _should_ be plenty in most cases but we should err on the side of a longer expiry.

Reply:
👍 LSM is an acronym from a previous life...